Source/BugsnagConfiguration.m

```diff
@@ -520,22 +514,6 @@ - (void)setReleaseStage:(NSString *)newReleaseStage {
 
 // MARK: -
 
-@synthesize autoDetectErrors = _autoDetectErrors;
-
-- (BOOL)autoDetectErrors {
-    return _autoDetectErrors;
-}
-
-- (void)setAutoDetectErrors:(BOOL)autoDetectErrors {
-    if (autoDetectErrors == _autoDetectErrors) {
-        return;
-    }
-    [self willChangeValueForKey:NSStringFromSelector(@selector(autoDetectErrors))];
-    _autoDetectErrors = autoDetectErrors;
-    [[Bugsnag client] updateCrashDetectionSettings];
-    [self didChangeValueForKey:NSStringFromSelector(@selector(autoDetectErrors))];
-}
-
```

Comment:
Is this removing the getter/setting altogether? This still needs to be settable by the user to pass it to `start`.

Also, should we remove the `autoNotify` backwards compatible property below?

Reply:
The property is declared on the header and Objective-C will automatically generate accessors for it. This is just overriding those accessors to include side-effects which are no longer required, so this won't have any impact on users being able to set the value.

I'll remove the `autoNotify` methods as they're not required.